app/services/meeting_starter.rb

```diff
@@ -57,7 +57,7 @@ def call
 
   def computed_options(access_code:)
     room_url = "#{root_url(host: @base_url)}rooms/#{@room.friendly_id}/join"
-    moderator_message = "#{I18n.t('meeting.moderator_message')}<br>#{room_url}"
+    moderator_message = "#{I18n.t('meeting.moderator_message', locale: @current_user.language.to_sym)}<br>#{room_url}"
     moderator_message += "<br>#{I18n.t('meeting.access_code', code: access_code)}" if access_code.present?
```

Comment:
Need to add it here as well

Reply:
I missed that thanks updated and pushed now!